modules/hcp-ts-load-balancer-svcs.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ts-load-balancer-svcs_{context}"]
+= Load balancer services for the hosted cluster are not available
+
+If a hosted control plane is not coming fully online because the load balancer services are not becoming available, check events, details, and the kccm pod.
```

Comment:
Should 'kccm' be expanded for the first use here? Although I see uses of the term without expansion in other areas of the OCP docs, so might be well-known in this space.

Reply:
I'll expand it to 'Kubernetes Cluster Configuration Manager' on the first mention.